docs/conf.py

```diff
@@ -76,7 +56,6 @@ def setup(sphinx):
     'github_issues_repository': 'scylladb/care-pet',
     'github_repository': 'scylladb/care-pet',
     'hide_edit_this_page_button': 'false',
-    'hide_sidebar_index': 'true',
```

Comment:
why removing this?

Reply:
See https://github.com/scylladb/sphinx-scylladb-theme/issues/229